doc/whats_new.rst

```diff
@@ -24,6 +24,10 @@ Changelog
 
     - Add system config utility :func:`mne.sys_info` by `Eric Larson`_
 
+    - Automatic cross-validation and scoring metrics in :func:`mne.decoding.GeneralizationAcrossTime`, by `Jean-Remi King`_
+
+    - :func:`mne.decoding.GeneralizationAcrossTime` accepts non-deterministic cross-validations, by `Jean-Remi King`_
```

Comment:
what do you mean by `non-deterministic`? This page should be easy to understand for the user